hack/kind/setup.sh

```diff
@@ -77,11 +77,21 @@ setup_cluster() {
   kubectl wait --for=condition=Ready pods --all --all-namespaces --timeout=300s
 }
 
+install_kubectl() {
+    if ! command -v kubectl &> /dev/null
+then
+    info "kubectl not found, attempting to install"
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
nice! 
Where would this install kubectl to?

Reply:
`kubectl` gets installed in the repo root with this. Hence the `PATH` addition (which I now also remembered to push).

I opted for this for now since I run this automated and it's nice if a script can just get its own dependency. I'd be up for a check before too, though I'd prefer to not have it cough up an error, require a manual step and then another run...at least not when the remedy is pretty straight forward and should work on all linux systems.